src/python/foglamp/admin_api/app_builder.py

```diff
@@ -0,0 +1,24 @@
+from aiohttp import web
+from .model import User
+from .login import register_handlers as login_register_handlers, auth_middleware
```

Comment:
1 import per line; 
Also I recommend here to use:
`from foglamp.admin_api  import login`
and then use 

```
app = web.Application(middlewares=[login.auth_middleware])
...

login.register_handlers(router) 
```